src/calculator.js

```diff
@@ -1,47 +1,33 @@
-exports._check = () => {
+exports._check = (x,y) => {
   // DRY up the codebase with this function
   // First, move the duplicate error checking code here
   // Then, invoke this function inside each of the others
   // HINT: you can invoke this function with exports._check()
-};
-
-exports.add = (x, y) => {
   if (typeof x !== 'number') {
-    throw new TypeError(`${x} is not a number`);
+    throw new TypeError('${x} is not a number');
   }
   if (typeof y !== 'number') {
-    throw new TypeError(`${y} is not a number`);
+    throw new TypeError('${y} is not a number');
```

Comment:
Unexpected template string expression.

###### See http://eslint.org/docs/rules/no-template-curly-in-string for details about this rule